### DISCUSSION TITLE
[RFC] 042 - Minimax Provider 集成

### DISCUSSION THREAD

**Discussion by arvinxx** [lobehub/lobehub#1959] [RFCs]
## 背景

用户提出希望集成 minimax 的模型，且有足够多的用户 upvote 了 request

- #1053

## 设计思路

Minimax 文档地址： https://www.minimaxi.com/document/guides/chat-model/V2?id=65e0736ab2845de20908e2dd

V2版本接口基本对齐了 openai 的出入参，但是比较坑爹的是 url 路径没有匹配上，因此不能直接使用 openai 包，请求部分需要自行构造。

不过整体对齐的情况下，复杂度应该不算高。

同时由于在 #1916 中大范围重构了 provider 的接入实现，因此需要一个 case 用于指导新模式下的 provider 集成。 (新模式下的代码关注部分应该大大缩小了)

## 进展

被大佬抢先一步： #2087

**Comment by MapleEve** [comment]
https://github.com/lobehub/lobe-chat/pull/2087

已经有大佬PR了 -。-
